ui/lib/kv/addon/components/kv-data-fields.hbs

```diff
@@ -15,7 +15,7 @@
 {{#if @showJson}}
   <JsonEditor
     @title="{{if (eq @type 'create') 'Secret' 'Version'}} data"
-    @value={{this.codeMirrorString}}
+    @value={{this.secretDataForCodeMirror}}
```

Comment:
nit: renaming this something like `stringifiedSecretData` would be a little more indicative of why a getter is used for this prop

Reply:
You know me and naming. Do not go to me first.